dev/build/windows/makecoq_mingw.sh

```diff
@@ -1275,7 +1275,6 @@ function get_cygwin_mingw_sources {
 
 function make_coq_installer {
   make_coq
-  make_mingw_make
```

Comment:
Didn't you forget to remove dead code in the script? In fact, that part broke recently.

Reply:
According to Michael this code may be useful in the future, and I'd rather leave it there and eventually disable it via the command line (or build `make.exe`, but not ship it)